File: Lab7-ListDemo/Lab7-ListDemo/Lab7-ListDemo/TableViewDemoVC/MyTableViewCell/MyTableViewCell.m
//
//  MyTableViewCell.m
//  Lab7-ListDemo
//
//  Created by 張力元 on 2019/3/6.
//  Copyright © 2019 張力元. All rights reserved.
//

#import "MyTableViewCell.h"

@interface MyTableViewCell()
@property (weak, nonatomic) IBOutlet UIImageView *myImageView;
@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
@end

@implementation MyTableViewCell
- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];
}

- (void)setCellImage:(NSString *)imageName title:(NSString *)title{
    [_myImageView setImage:[UIImage imageNamed:imageName]];
    _titleLabel.text = title;
}
@end

File: Swift/lab7/lab7_exercise/lab7_exercise/tableViewDemoVC/myTableViewCell/myTableViewCell.swift
//
//  myTableViewCell.swift
//  lab7-1-2
//
//  Created by taizhou on 2019/7/29.
//  Copyright © 2019 taizhou. All rights reserved.
//

import UIKit

class myTableViewCell: UITableViewCell {
    @IBOutlet var myImgView: UIImageView!
    @IBOutlet var titleLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
    func setCell(imgName:String,title:String){
        myImgView.image = UIImage(named: imgName)
        titleLabel.text = title
    }
}
